src/js/components/Footer/stories/Sitemap.js

```diff
@@ -38,8 +40,8 @@ export const Sitemap = () => (
     <Footer background="dark-1" pad="large">
       <Box direction="row-responsive" gap="xsmall">
         <Box align="center" gap="small">
-          <GrommetIcon color="brand" size="large" />
-          <Text alignSelf="center" color="brand" weight="bold">
+          <GrommetIcon color="white" size="large" />
+          <Text alignSelf="center" color="white" weight="bold">
```

Comment:
```suggestion
          <GrommetIcon size="large" />
          <Text alignSelf="center" weight="bold">
```
We should be able to not set a color here and Grommet will detect that the text and icon are on a dark background and set them to be white automatically.

Reply:
I see, I'll try to fix this and get back to you.